content-src/experiments/color.yaml

```diff
@@ -0,0 +1,76 @@
+id: 17
+title: 'Color'
+slug: color
+is_featured: false
+platforms: ['addon', 'web']
+min_release: 60
+thumbnail: /static/images/experiments/color/color-thumbnail.png
+description: >
+  Show your colors. Create and share your own Firefox themes with just a few clicks.
+introduction: >
+  FIrefox Color makes it easy for anyone to change the look of Firefox.
+  It’s a simple tool that lets you choose different color combinations
+  and background textures for your own Firefox theme.
+image_twitter: /static/images/experiments/notes/notes-twitter.jpg
+image_facebook: /static/images/experiments/notes/notes-fb.jpg
```

Comment:
probably just placeholders, but these still reference "notes-fb" and "notes-twitter".

Reply:
Yep see #3562